src/test/java/edu/cornell/kfs/concur/businessobjects/ValidationResultTest.java

```diff
@@ -0,0 +1,54 @@
+package edu.cornell.kfs.concur.businessobjects;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ValidationResultTest {
+    
+    private ValidationResult validationResult;
+    
+    private static final String TEST_STRING_ONE = "test string 1";
+    private static final String TEST_STRING_TWO = "test string 2";
+    private static final String TEST_STRING_THREE = "test string 3";
```

Comment:
As a follow-up to the question @nkimble had about case sensitivity, it might be helpful to have some mixed-case values to test with in here.